<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Информация о предприятии</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_full_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Полное наименование</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сокращенное наименование</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_short_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Юридический адрес</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_adress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ИНН</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес электронной почты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_site">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сайт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_boss">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Иванов Петр Тимофеевич</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО руководителя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_boss_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность руководителя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_metrolog">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Пронин Сергей Яковлевич</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО ответственного за метрологию</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_metrolog_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность ответственного за метрологию</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code_mark">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Шифр знака поверки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_accred_number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Уникальный номер в реестре аккредитованных лиц</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
